domains/game/Game.jsx

```diff
@@ -1,26 +1,105 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Text, View, Button, StyleSheet } from "react-native";
 import PropTypes from "prop-types";
 
+const obstacles = [
+    { x: 0, y: 3 },
+    { x: 1, y: 1 },
+    { x: 1, y: 5 },
+    { x: 2, y: 2 },
+    { x: 2, y: 4 },
+    { x: 3, y: 0 },
+    { x: 3, y: 6 },
+    { x: 4, y: 2 },
+    { x: 4, y: 4 },
+    { x: 5, y: 1 },
+    { x: 5, y: 5 },
+    { x: 6, y: 3 },
+];
+
 const Game = ({ playerName, game }) => {
+    const renderRow = (rowIndex) => {
+        const cells = [];
+        for (let index = 0; index < 7; index++) {
```

Comment:
It would be best to generate a two dimensions array from obstacles, and then map over it during render.